src/App.js

```diff
@@ -105,7 +112,7 @@ function App() {
         <BrowserRouter>
           <Routes>
             <Route path="/" element={
-              <>
+              <div className={styles.container}>
```

Comment:
This isn't actually doing anything because `styled-components` doesn't work like a css module. Instead of accessing your `style` object and applying a specific selector, it allows you to create a styled component right in your `.js` file. To make it work with the css module and styling you've already created, edit it to the following:
```
<div className={style.container}>
```

To use `styled-components` properly, edit the import (as I mentioned in a previous comment) and above the declaration of the `App` function add something like the following:

```
const Container = styled.div`
  display: flex;
  flex-direction: column;
  align-items: center;
  color: #1c1f23;
`;
```
And then edit that `<div>` into a `<Container>` with the styled component.

I pulled those styles directly from your `App.module.css` as those styles aren't being implemented due to a faulty import statement.